### PR TITLE
PIN-4541 Add update of lifespan and daily calls on descriptors

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -460,6 +460,57 @@ paths:
...
+  /eservices/{eServiceId}/descriptors/{descriptorId}/update:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Publish the selected descriptor.
+      operationId: updatePublishedDescriptor
```
Comment (galales) [inline src/main/resources/interface-specification.yml]:
```suggestion
      operationId: updateDescriptor
```

File: src/main/resources/interface-specification.yml [hunk 2]
```diff
@@ -460,6 +460,57 @@ paths:
...
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Payload
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateEServicePublishedDescriptorSeed'
```
Comment (galales) [inline src/main/resources/interface-specification.yml]:
```suggestion
              $ref: '#/components/schemas/UpdateEServiceDescriptorSeed'
```

File: src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala [hunk 3]
```diff
@@ -475,6 +475,39 @@ final case class ProcessApiServiceImpl(
...
+    eServiceId: String,
+    descriptorId: String,
+    seed: UpdateEServicePublishedDescriptorSeed
+  )(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerEService: ToEntityMarshaller[EService]
+  ): Route =
+    authorize(ADMIN_ROLE, API_ROLE) {
+      val operationLabel = s"Update Published Descriptor $descriptorId for EService $eServiceId"
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala]:
```suggestion
      val operationLabel = s"Update Descriptor $descriptorId for EService $eServiceId"
```

File: src/test/scala/it/pagopa/interop/catalogprocess/CatalogProcessSpec.scala [hunk 4]
```diff
@@ -1710,6 +1710,246 @@ class CatalogProcessSpec extends SpecHelper with AnyWordSpecLike with ScalatestR
       }
     }
   }
+  "Published Descriptor update" should {
```
Comment (galales) [inline src/test/scala/it/pagopa/interop/catalogprocess/CatalogProcessSpec.scala]:
```suggestion
  "Descriptor update" should {
```

File: src/test/scala/it/pagopa/interop/catalogprocess/authz/ProcessApiAuthzSpec.scala [hunk 5]
```diff
@@ -129,6 +129,15 @@ class ProcessApiAuthzSpec extends AnyWordSpecLike with BeforeAndAfterAll with Au
       )
     }
 
+    "accept authorized roles for updatePublishedDescriptor" in {
```
Comment (galales) [inline src/test/scala/it/pagopa/interop/catalogprocess/authz/ProcessApiAuthzSpec.scala]:
```suggestion
    "accept authorized roles for updateDescriptor" in {
```

**Review by galales — COMMENTED** [review]
LGTM, minor renaming requested

File: src/main/resources/interface-specification.yml (hunk 2 above)
Comment (nttdata-rtorsoli) [reply src/main/resources/interface-specification.yml]: Already exists are you sure?

Comment (galales) [reply src/main/resources/interface-specification.yml]: you're right, let's find another name

Comment (beetlecrunch) [reply src/main/resources/interface-specification.yml]: `UpdateEServiceDescriptorQuotas`?
`UpdateEServiceDescriptorDetails`?

**Review by galales — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**